File: app/src/main/java/omniex/nl/omniex/ui/app/main/home/HomeView.java
package omniex.nl.omniex.ui.app.main.home;

import java.util.List;

import nl.omniex.omniexshopping.data.model.products.FeaturedProduct;
import nl.omniex.omniexshopping.ui.base.BaseView;

public interface HomeView extends BaseView {

    void onFeaturedProductsFetched(List<FeaturedProduct> featuredProducts);
}

File: app/src/main/java/omniex/nl/omniex/ui/app/main/home/HomeView.kt
package omniex.nl.omniex.ui.app.main.home

import nl.omniex.omniexshopping.data.model.products.FeaturedProduct
import nl.omniex.omniexshopping.ui.base.BaseView

interface HomeView : BaseView {

    fun onFeaturedProductsFetched(featuredProducts: List<FeaturedProduct>)
}
